src/main/scala/nl/knaw/dans/easy/fedora2vault/package.scala

```diff
@@ -30,7 +30,9 @@ package object fedora2vault {
 
   val dateTimeFormatter: DateTimeFormatter = ISODateTimeFormat.dateTime()
 
-  def now: String = DateTime.now(DateTimeZone.UTC).toString(dateTimeFormatter)
+  def now: String = DateTime.now(DateTimeZone.UTC)
+    .toString(dateTimeFormatter)
+    .replaceAll("[:.]","-")
```

Comment:
This is fine as long as `now` is only used in `CommandLineOptions`. It would probably be less error prone if you move this `replaceAll` to `CommandLineOptions` itself.